Lib/modulefinder.py

```diff
@@ -25,6 +25,7 @@
 _PKG_DIRECTORY = 5
 _C_BUILTIN = 6
 _PY_FROZEN = 7
+_NSP_DIRECTORY = 8
```

Comment:
`_NS_PKG_DIRECTORY`? Perhaps not.

Reply:
I believe that was indeed my first draft, but I decided to shorten it. 
This is an internal-only enum used at very few locations so it did not matter so much to me.